File: app/src/main/java/bj/rxjavaexperimentation/gson/Community.java

package bj.rxjavaexperimentation.gson;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Community {

    @SerializedName("have")
    @Expose
    private Integer have;
    @SerializedName("want")
    @Expose
    private Integer want;

    public Integer getHave() {
        return have;
    }

    public void setHave(Integer have) {
        this.have = have;
    }

    public Integer getWant() {
        return want;
    }

    public void setWant(Integer want) {
        this.want = want;
    }

}

File: app/src/main/java/bj/discogsbrowser/model/release/Community.kt
package bj.discogsbrowser.model.release

import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Community(var contributors: List<Contributor> = emptyList(),
                     @SerializedName("data_quality") var dataQuality: String = "",
                     var have: Int = 0,
                     var rating: Rating = Rating(0.0, 0),
                     var status: String = "",
                     var submitter: Submitter = Submitter("", ""),
                     var want: Int = 0)